source/Port.h

```diff
@@ -0,0 +1,104 @@
+/* Port.h
+Copyright (c) 2021 by Michael Zahniser
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef PORT_H_
+#define PORT_H_
+
+#include <string>
+
+class DataNode;
+
+
+
+// Class representing a port on a planet.
```

Comment:
```suggestion
// Class representing a port on a planet and its capabilities, such as what properties
// of a ship it can recharge and what services (e.g. banking, trading) it can provide.
```